sink/datadog.go

```diff
@@ -277,8 +279,23 @@ func (s *Datadog) Send(ctx context.Context, m *blip.Metrics) error {
 			// Convert Blip metric type to Datadog metric type
 			switch metrics[i].Type {
 			case blip.COUNTER:
+				// calculate delta
+				val, ok := s.counters[name]
```

Comment:
Good point @Areson .

Both `size.database` and `size.table` are `guage` metrics so they won't be affected by this.

`query.response-time` is a `guage` as well.

However, `wait.io.table` is a counter. It's also a common use case we should handle.

I was not taking `tag` aspect of the metrics into consideration at all, my bad. With current implementation, it would alter/override values for different tags.

I can see we are building a `tags` array, I can probably just append the contents of that into the name to get a unique ID for the metric/tags combination? I will give it a go.